kivy/lang/parser.py

```diff
@@ -422,6 +422,10 @@ def execute_directives(self):
                     ref = ref[6:].strip()
                     force_load = True
 
+                # if #:include [force] "path with quotes around"
+                if ref[0] == ref[-1] and ref[0] in ('"', "'"):
+                    ref = ref[1:-1]
```

Comment:
What about triple quotes :D
Although to be honest supporting triple quotes is probably not that important.